infra/ansible/inventories/production/hosts

```diff
@@ -26,11 +26,21 @@ all:
                       name: airqo-k8s-worker-4                
     mongo_cluster:
       hosts:
-        mongos-router.europe-west1-b.c.airqo-250220.internal
+        mongos_router:
+          ansible_host: mongos-router.europe-west1-b.c.airqo-250220.internal
       children:
         config_servers:
           hosts:
-            mongod-cfgsvr-[1:3].europe-west1-b.c.airqo-250220.internal
-        shards_1:
+            mongod-cfgsvr-0.europe-west1-b.c.airqo-250220.internal:
+            mongod-cfgsvr-1.europe-west1-c.c.airqo-250220.internal:
+            mongod-cfgsvr-2.europe-west1-d.c.airqo-250220.internal:
+        shard_0:
           hosts:
-            mongod-shard-1-[1:3].europe-west1-b.c.airqo-250220.internal
\ No newline at end of file
+            mongod-shard-0-0.europe-west1-b.c.airqo-250220.internal:
+            mongod-shard-0-1.europe-west1-c.c.airqo-250220.internal:
+            mongod-shard-0-2.europe-west1-d.c.airqo-250220.internal:
+        shard_1:
+          hosts:
+            mongod-shard-1-0.europe-west1-b.c.airqo-250220.internal:
+            mongod-shard-1-1.europe-west1-c.c.airqo-250220.internal:
+            mongod-shard-1-2.europe-west1-d.c.airqo-250220.internal:
```

Comment:
[add shard_1 to hosts](https://github.com/airqo-platform/AirQo-api/pull/1538/commits/edd14d103956eb85bb0bd0ce3bea4becd696ac7f)